README.md

```diff
@@ -1,5 +1,9 @@
 # vscode-opa
 
+To install the extension, visit the
+[Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=tsandall.opa)
+or search for "Open Policy Agent" in the 'Extensions' panel.
```

Comment:
I'd place this under `Features`, and add a `Installation` heading, so that we start off by presenting what this actually is before telling users how to install it.

Also, perhaps we should add instructions for installing Regal, or just a link to the Regal docs on that? It's a big part of this extension's experience now.

Reply:
Yeah this was just a what, there is no link reaction! I have made second attempt now, lmk if that looks better.